src/main/java/software/amazon/cloudformation/LambdaWrapper.java

```diff
@@ -570,6 +571,15 @@ protected void scrubFiles() {
         return previousResourceTags;
     }
 
+    @VisibleForTesting
+    protected String getStackId(final HandlerRequest<ResourceT, CallbackT> request) {
+        if (request != null) {
+            return request.getStackId();
+        }
+
+        return "";
```

Comment:
null maybe?

Reply:
should we add this to all plugins @wbingli ?

Reply:
Yes, we should